configure.ac

```diff
@@ -46,5 +46,12 @@ AC_TRY_LINK([
 AC_DEFINE(HAVE_FULL_QUEUE_H,, Define if sys/queue.h includes required macros)
 ])
 
+AC_TRY_LINK([
+	#include <ncursesw/curses.h>
+	#include <ncursesw/term.h>
+], [], [
+AC_DEFINE(HAVE_NCURSESW_H,, Define if ncursesw is available)
+])
```

Comment:
Yeap, it builds (it's also a [patch](https://github.com/eavgerinos/pkg-pick/blob/master/debian/patches/ncursesw-compatibility) I included in the debian package).

The [`AC_TRY_LINK`](http://ftp.gnu.org/old-gnu/Manuals/autoconf-2.53/html_node/Examining-Libraries.html) defines the `HAVE_NCURSESW_H` hdef.

On the other hand [`AC_SEARCH_LIBS`](https://www.gnu.org/software/autoconf/manual/autoconf-2.69/html_node/Libraries.html) iterates through a list of alternative choices in order to tell if at least 1 of the alternatives is available on the system.

So, imho, we need `AC_TRY_LINK`